eng/common/scripts/TypeSpec-Project-Sync.ps1

```diff
@@ -103,7 +103,9 @@ $specSubDirectory = $configuration["directory"]
 
 # use local spec repo if provided
 if ($LocalSpecRepoPath) {
-  $specCloneDir = $LocalSpecRepoPath
+  $LocalSpecRepoPath = $LocalSpecRepoPath.Replace("\", "/")
+  # remove duplicated 'specification' in path
+  $specCloneDir = $LocalSpecRepoPath.Replace("/specification","")
```

Comment:
What cases do we have where there is duplicate "/specification" in the path? Should we fix those instead of adding this here? If we have a good reason we cannot fix the other places then lets add a comment here about why we need to do this.